guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_prerequisites.adoc

```diff
@@ -9,6 +9,9 @@ For more information, see {InstallingProjectDocURL}preparing-environment-for-sat
 * Ensure that you do not delete the manifest from the Customer Portal or in the {Project} Web UI because this removes all the entitlements of your content hosts.
 * Back up and remove all Foreman hooks before upgrading.
 Restore any hooks only after {Project} is known to be working after the upgrade is complete.
++
+Note that Foreman Hooks functionality is deprecated and will be removed in the next {Project} version.
```

Comment:
No need for the blank lines before and after this.

Reply:
Changes made.

Reply:
As per Ewoud's feedback, wrap this in a Satellite ifdef statement.

Reply:
Changed as per comment